From: Markus Blatt <markus@dr-blatt.de>
Date: Tue, 8 Nov 2016 20:43:38 +0100
Subject: Use flags recommended by dpkg-buildflags for building

This should prevent the lintian warning about hardening-no-relro
---
 debian/rules | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index c816b38..4d1fcdd 100755
--- a/debian/rules
+++ b/debian/rules
@@ -9,8 +9,14 @@
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
+CMAKE_C_FLAGS="$(shell  dpkg-buildflags --get CFLAGS) $(shell  dpkg-buildflags --get CPPFLAGS)"
+CMAKE_CXX_FLAGS="$(shell  dpkg-buildflags --get CFLAGS) $(shell  dpkg-buildflags --get CPPFLAGS)"
+CMAKE_SHARED_LINKER_FLAGS="$(shell  dpkg-buildflags --get LDFLAGS)"
 %:
 	dh $@
 
 override_dh_auto_configure:
-	DESTDIR=$$(pwd)/debian/tmp dh_auto_configure -- -DBUILD_SHARED_LIBS=1 -DBUILD_ECL_SUMMARY=1 -DCMAKE_BUILD_TYPE=Release
+	DESTDIR=$$(pwd)/debian/tmp dh_auto_configure -- -DBUILD_SHARED_LIBS=1 \
+           -DBUILD_ECL_SUMMARY=1 -DCMAKE_C_FLAGS=$(CMAKE_C_FLAGS) \
+           -DCMAKE_CXX_FLAGS=$(CMAKE_CXX_FLAGS) \
+           -DCMAKE_SHARED_LINKER_FLAGS=$(CMAKE_SHARED_LINKER_FLAGS)
